src/styles/addToAlbum.styl

```diff
@@ -7,3 +7,7 @@
     .coz-modal-section
         border-top 1px solid grey-09
         padding    1em 1.5em .5em !important
```

Comment:
Oh fuck me. WTF is that `!important` doing here? 😠
I know it's not from you, at least not in this PR, but i needed to say this.